backend/internal/system/constants/serverconstants.go

```diff
@@ -43,3 +43,9 @@ const ContentTypeJSON = "application/json"
 
 // ContentTypeFormURLEncoded is the content type for form-urlencoded data.
 const ContentTypeFormURLEncoded = "application/x-www-form-urlencoded"
+
+// DefaultPageSize is the default limit for pagination when not specified.
+const DefaultPageSize = 30
```

Comment:
Could there be a requirement to have a service level default values for paginated params?

Reply:
Lets keep it simple and improve if the requirement comes up.